src/systempolicy/systemPolicy.go

```diff
@@ -920,6 +956,11 @@ func PopulateSystemPoliciesFromSystemLogs(sysLogs []types.KnoxSystemLog) []types
 					polCnt = len(discoveredSysPolicies)
 					log.Info().Msgf("discovered %d file policies from %d file logs",
 						len(discoveredSysPolicies), len(fileOpLogs))
+				} else {
+					// New mode of system policy generation using WPFS table
+					if isWpfsDbUpdated {
```

Comment:
Lets have single isWpfsDbUpdated check across file/process/network and then do updateSysPolicies() in it.